src/routes/tutorial/[slug]/Chrome.svelte

```diff
@@ -23,6 +23,12 @@
 		on:change={(e) => {
 			dispatch('change', { value: e.currentTarget.value });
 		}}
+		on:keydown={(e) => {
+			if (e.key === 'Enter') {
+				dispatch('change', { value: e.currentTarget.value });
+				e.currentTarget.blur();
```

Comment:
Without this, the focus will remain on the input.
It should focus on the iframe, but I have no idea of a better way to do it, so I did it this way.